lib/main.js

```diff
@@ -76,6 +78,11 @@ var EVENT={
     error:'error'
 }
 
+function anonymizeMode(flag) {
+    anonymizationEnabled = flag;
```

Comment:
The variable `anonymizationEnabled` should not be under the scope of main.js file. You can encapsulate the anonymization toggle(true|false) inside the anonymizer.js, and where needed, to call a function from anonymizer.js to get if the anonymization is on or off. eg. anonymizer.isOn could return true or false

Reply:
ok